CONTRIBUTING.md

```diff
@@ -37,6 +39,10 @@ and now we can run the local Jupyter Lab and run the scripts etc. using Poetry:
 poetry run jupyter lab
 ```
 
+!!! note
+
+    You may need to use `poetry run make ...` to run commands in the environment.
+
```

Comment:
Your original syntax is not valid. You want this:

```suggestion
> [!NOTE]
> You may need to use `poetry run make ...` to run commands in the environment.
```

Reply:
Sorry, scratch that. The syntax is invalid for GitHub Markdown, which is what my "correction" addresses, but I see that your syntax is for the generated docs.

It's a shame that this does not also render properly on GitHub. Do we need to determine how to cleanly separate such things?

Reply:
Yes, this is one of my pet peeves. Same with Quarto, admonitions are rendered as

```
:::{.callout-note}
My note
:::
```

And Sphinx:

```
.. note::

   My note
```

I really dislike how GitHub invented their own syntax for this on top of the fifty we already have.

Reply:
I think we should make `/CONTRIBUTING.md` simply a link to a `docs/contributing/index.md`. Could do that as part of this PR to resolve this. What do you think?

Reply:
8de415d

Reply:
I didn't think about how much harder this would make review. Apologies.